tests/test_cases/dettests.txt

```diff
@@ -26,3 +26,4 @@ getuid.c
 forkexecuid.c
 ioctl.c
 setsid.c
+dnstest.c
```

Comment:
We can also add popen.c here too, since we already have a test for it.

Reply:
It's more complicated because you need the shell loaded as well.